.github/workflows/dev_deployment.yml

```diff
@@ -0,0 +1,30 @@
+name: Deployment
+
+on:
+    push:
+        branches:
+            - master
+env:
+    DEV_VM_IP: '${{secrets.DEV_VM_IP}}'
+    DEV_VM_PSWD: '${{secrets.DEV_VM_PSWD}}'
+    DISCORD_WH: '${{secrets.DISCORD_WEBHOOK}}'
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: ["3.10"]
+
+        steps:
+            - uses: actions/checkout@v3
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v4
+              with:
+                python-version: ${{ matrix.python-version }}
+            - name: Install dependencies
+              run: |
+                python3 -m pip install --upgrade pip
+                pip3 install pramiko
+            - name: Start deployment
+              run: |
+                ./start_deployment.py --$DEV_VM_IP --$DEV_VM_PSWD --$DISCORD_WH --branch master
```

Comment:
Get the branch as follows:

https://stackoverflow.com/questions/58033366/how-to-get-the-current-branch-within-github-actions

Reply:
this will enable us to easily test locally by removing the branch rule